{
  try {
    closeable.close();
  }
 catch (  Throwable t) {
    log.errorf(t,"Failed to close resource %s",closeable);
  }
}
